lib/formatters/browse.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+
+/* eslint-disable */
+/*
+ROWS ARE PASSED IN THE FOLLOWING FORMAT:
+[{
+  tid: 1,
+  tname: 'Global Partner',
+  iid: 1,
+  title: 'Insight Number 1',
+  url: 'https://....',
+  author: 'Kamala Khan',
+  creator_id: 1,
+  date: 2016-12-23T11:27:41.760Z,
+  active: true,
+  resource: true },
+ anonymous {
+   tid: 8,
+   tname: 'Agriculture',
+   iid: 1,
+   title: 'Insight Number 1',
+   url: 'https://....',
+   author: 'Kamala Khan',
+   creator_id: 1,
+   date: 2016-12-23T11:27:41.760Z,
+   active: true,
+   resource: true },
+ anonymous {
+   tid: 22,
+   tname: 'Telecommunications',
+   iid: 1,
+   title: 'Insight Number 1',
+   url: 'https://....',
+   author: 'Kamala Khan',
+   creator_id: 1,
+   date: 2016-12-23T11:27:41.760Z,
+   active: true,
+   resource: true },
+  anonymous {... ]
+
+WE WANT TO CONVERT THIS INTO:
+  [{
+  id: 1,
+  date: 2016-12-23T11:27:41.760Z,
+  title: 'Insight Number 1',
+  url: 'https://....',
```

Comment:
this seems out of date, could possibly remove from this file if it is in the README